tasks/eth/008-fp-upgrade/VALIDATION.md

```diff
@@ -0,0 +1,62 @@
+
+Links:
+- [Etherscan](https://etherscan.io/address/0x229047fed2591dbec1eF1118d64F7aF3dB9EB290)
+- [Superchain Registry](https://github.com/ethereum-optimism/superchain-registry/blob/94149a2651f0aadb982802c8909d60ecae67e050/superchain/extra/addresses/mainnet/op.json#L10)
+
+State Changes:
+- **Key:** `0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc` <br/>
+  **Before:** `0x000000000000000000000000ba2492e52f45651b60b8b38d4ea5e2390c64ffb1` <br/>
+  **After:** `0x000000000000000000000000f56d96b2535b932656d3c04ebf51babff241d886` <br/>
+  **Meaning:** This upgrades the SystemConfig implementation. Verify that the new `SystemConfig` implementation is stoored at the eip1967 proxy implementation slot.
+
+- **Key**: `0x52322a25d9f59ea17656545543306b7aef62bc0cc53a0e65ccfa0c75b97aa906`
+  **Before**: `0x0000000000000000000000000000000000000000000000000000000000000000`
+  **After**: `0x000000000000000000000000e5965ab5962edc7477c8520243a95517cd252fa9`
+  **Meaning**: Sets the [DisputeGameFactory slot](https://github.com/ethereum-optimism/optimism/blob/op-contracts/v1.4.0-rc.4/packages/contracts-bedrock/src/L1/SystemConfig.sol#L76). Verify that the `Key` matches the expected slot.
+
+- **Key**: `0xe52a667f71ec761b9b381c7b76ca9b852adf7e8905da0e0ad49986a0a6871815`
+  **Before**: `0x000000000000000000000000dfe97868233d1aa22e815a266982f2cf17685a27`
+  **After**: `0x0000000000000000000000000000000000000000000000000000000000000000`
+  **Meaning**: Clears the old and unused [L2OutputOracle slot](https://github.com/ethereum-optimism/optimism/blob/op-contracts/v1.3.0/packages/contracts-bedrock/src/L1/SystemConfig.sol#L63). Verify that the `Key` matches the expected slot.
+
+
+### `0xbeb5fc579115071764c7423a4f12edde41f106ed` (`OptimismPortalProxy`)
+
+Links:
+- [Etherscan](https://etherscan.io/address/0xbEb5Fc579115071764c7423A4f12eDde41f106Ed)
+- [Superchain Registry](https://github.com/ethereum-optimism/superchain-registry/blob/52d3dbd1605dd43f419e838584abd0ec163d462b/superchain/extra/addresses/mainnet/op.json#L8)
+
+Upgrades the `OptimismPortal` to the `OptimismPortal2` implementation.
+
+State Changes:
+- **Key:** [`0x0000000000000000000000000000000000000000000000000000000000000038`](https://github.com/ethereum-optimism/optimism/blob/op-contracts/v1.4.0-rc.4/packages/contracts-bedrock/snapshots/storageLayout/OptimismPortal2.json#L80C1-L85C5) <br/>
+  **Before**: `0x0000000000000000000000000000000000000000000000000000000000000000` <br/>
+  **After**: `0x000000000000000000000000e5965ab5962edc7477c8520243a95517cd252fa9` <br/>
+  **Meaning**: Sets the `DisputeGameFactoryProxy` address in the proxy storage. Consult the [gov proposal](https://gov.optimism.io/t/final-protocol-upgrade-7-fault-proofs/8161) for the proxy addreess value.
+One can verify that this change was made by running `cast call 0xbeb5fc579115071764c7423a4f12edde41f106ed 'disputeGameFactory()(address)' https://ethereum.publicnode.com`
```

Comment:
This won't work until after the ceremony right? So, I don't think it should be part of the signing validations.

Reply:
That's correct, especially the simulated part. The ways we currently do the simulation are:
1. in memory execution with `forge script`
2. tenderly simulation

But the `cast call` approach would require an rpc endpoint with the simulated state, which we don't currently have. 

I think you can just remove that though, the other content for validating this change is consistent with what we've done in the past.